src/AWS.Deploy.Common/DeploymentManifest/DeploymentManifestModel.cs

```diff
@@ -11,11 +11,14 @@ namespace AWS.Deploy.Common.DeploymentManifest
     /// </summary>
     public class DeploymentManifestModel
     {
+        public List<LastDeployedStack> LastDeployedStacks { get; set; }
+
         [JsonProperty("deployment-projects")]
         public List<DeploymentManifestEntry> DeploymentManifestEntries { get; set; }
```

Comment:
I'm seeing when I use this that `deployment-projects` is getting written to the JSON file as an empty string. I'm assuming `LastDeployedStacks` would also be serialized as an empty array as well if there were no values. Malhar did some work in his previous PR to make sure we weren't writing out empty lists when serializing the recipe. Check with him to see what he did.

Reply:
Fixed